modules/sdk-coin-sol/src/sol.ts

```diff
@@ -119,6 +120,8 @@ interface RecoveryOptions {
     publicKey: string;
     secretKey: string;
   };
+  startingScanningIndex?: number;
```

Comment:
nit: `startingScanIndex`?

if its consistent with Dot + other coins, leave it as is